src/core/addonManager.js

```diff
@@ -123,15 +125,18 @@ export function install(
   return _mozAddonManager.createInstall({ url, hash }).then((installObj) => {
     const callback = (e) => eventCallback(installObj, e);
     for (const event of INSTALL_EVENT_LIST) {
-      log.info(`[install] Adding listener for ${event}`);
+      _log.info(`[install] Adding listener for ${event}`);
       installObj.addEventListener(event, callback);
     }
     return new Promise((resolve, reject) => {
       installObj.addEventListener('onInstallEnded', () => resolve());
       installObj.addEventListener('onInstallFailed', () => reject());
-      log.info('Events to handle the installation initialized.');
-      // See: https://github.com/mozilla/addons-frontend/issues/8633
-      installObj.install().catch(reject);
+      _log.info('Events to handle the installation initialized.');
+      installObj.install().catch((error) => {
+        // See: https://github.com/mozilla/addons-frontend/issues/8668
+        _log.warn(`Ignoring error during installation: ${error}`);
```

Comment:
Maybe instead of "ignoring error" you could say "ignoring promise rejection" because it seems to me that this is not actually an error (which could be a Firefox bug).

Reply:
A rejected promise is also an error but the phrasing will be more accurate with "promise rejection".

> which could be a Firefox bug

Yes. I noted in the other issue that I wanted to see with the FF team about it.